clusters/build-clusters/common_managed/rh-logging.yaml

```diff
@@ -60,20 +60,30 @@ spec:
       url: https://http-inputs-rhcorporate.splunkcloud.com
       secret:
         name: splunk-receiver-token
+    - name: cloudwatch-receiver
+      type: cloudwatch
+      cloudwatch:
+        groupBy: logType
+        region: us-east-1
+      secret:
+        name: cloudwatch-receiver-token
```

Comment:
Does it work in the local test?
I am not familiar with `ClusterLogForwarder`.

Reply:
I deployed it on b10 already